### PR TITLE
fix: do not show upgrade prompt for standups and checkins

### PR DESCRIPTION
**Pull request by BartoszJarocki**
# Description

Fixes showing upgrade prompt for activities like standups and checkins

## Demo

No demo

## Testing scenarios

- [ ] Create starter team, try to run standup
- [ ] Create starter team, try to run checkin
- [ ] Make sure there's still upgrade prompt showing for premium templates

## Final checklist

- [x] I checked the [code review guidelines](../docs/codeReview.md)
- [x] I have added [Metrics Representative](../docs/codeReview.md#metrics-representative) as reviewer(s) if my PR invovles metrics/data/analytics related changes
- [x] I have performed a self-review of my code, the same way I'd do it for any other team member
- [x] I have tested all cases I listed in the testing scenarios and I haven't found any issues or regressions
- [x] Whenever I took a non-obvious choice I added a comment explaining why I did it this way
- [x] I added the label `One Review Required` if the PR introduces only minor changes, does not contain any architectural changes or does not introduce any new patterns and I think one review is sufficient'
- [x] PR title is human readable and could be used in changelog

### CODE REVIEW

File: packages/client/components/ActivityLibrary/ActivityDetailsSidebar.tsx [hunk 1]
```diff
@@ -220,7 +220,7 @@ const ActivityDetailsSidebar = (props: Props) => {
             />
           )}
 
-          {selectedTeam.tier === 'starter' && !selectedTemplate?.isFree ? (
+          {selectedTeam.tier === 'starter' && selectedTemplate && selectedTemplate.isFree ? (
```
Comment (Dschoordsch) [inline packages/client/components/ActivityLibrary/ActivityDetailsSidebar.tsx:223]: -1 Fails, in a starter team I can see premium templates with a "Start activity" button. When starting it, it seems to work but the server prints `SEND TO SENTRY Error: User does not have access to this premium template`.
On master I see the upgrade button in this case. Tested with Team Charter and SWOT Analysis

Comment (BartoszJarocki) [reply packages/client/components/ActivityLibrary/ActivityDetailsSidebar.tsx]: I'm sorry, this is just a mistake from testing phase, I removed `!` from `!selectedTemplate.isFree` condition. could you double check now?

Comment (Dschoordsch) [reply packages/client/components/ActivityLibrary/ActivityDetailsSidebar.tsx]: Should have seen this :facepalm:

**Review by Dschoordsch — APPROVED** [review]

File: packages/client/components/ActivityLibrary/ActivityDetailsSidebar.tsx (hunk 1 above)
Comment (BartoszJarocki) [reply packages/client/components/ActivityLibrary/ActivityDetailsSidebar.tsx]: nah, that should not have happened! thank you for actually checking that.

**Pull request merged**